src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -489,23 +472,21 @@ def nonhermetic_digest_classpath():
               "-Ypickle-write",
               rsc_jar_file_relative_path,
             ]
-            if not self.get_options().allow_public_inference:
-              wartremover_args = [
-                f"-Xplugin:{self._wartremover_classpath[0]}",
-                "-P:wartremover:traverser:org.wartremover.warts.PublicInference",
-                "-Ycache-plugin-class-loader:last-modified",
-              ]
-              youtline_args = wartremover_args + youtline_args
 
           target_sources = ctx.sources
-          args = [
-                   '-cp', os.pathsep.join(classpath_entry_paths),
-                   '-d', rsc_jar_file_relative_path,
-                 ] + self.get_options().extra_rsc_args + youtline_args + target_sources
+          
+          # TODO: m.jar digests aren't found, so hermetic will fail.
```

Comment:
Stale, I presume?

Reply:
Nope, added as a reason why zinc outlining will not work under hermetic. However, the previous implementation will continue to work hermetically.